c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -907,8 +906,8 @@ static inline void reinjection_set_wait2_timeout(uint payload) {
 //! response
 //! \return The payload size of the response message.
 static inline int reinjection_set_timeout_sdp(sdp_msg_t *msg) {
-#if 0
-    io_printf(IO_BUF, "setting router timeouts via sdp\n");
+#ifdef DEBUG_REINJECTOR
```

Comment:
Again here I don't remember DEBUG_REINJECTOR being defined anywhere, though again I guess the user can choose to turn this on.  We should probably have an action build which turns these on at least to check the code inside them can be compiled, though.

Reply:
There's `DEBUG_REINJECTOR`, `DEBUG_DATA_IN` and `DEBUG_DATA_OUT`.